api/system-status/read

```diff
@@ -32,7 +32,45 @@ function readMemory()
     return $fields; 
 }
 
-function readUptime() {
+function readDisk()
+{
+    $out = array();
+    $ret = array();
+    exec('/bin/df -P /', $out);
+    # Filesystem Size  Used Avail Use% Mount
+    for ($i=0; $i<count($out); $i++) {
+        if ($i == 0) {
+            continue;
+        }
+        $tmp = explode(" ", preg_replace('/\s+/', ' ', $out[$i]));
+        // skip fs ($tmp[0]) and perc_used ($tmp[4])
+        if ($tmp[5] == "/") {
+          $tmp[5] = "root";
+        }
+        $ret[$tmp[5]] = array('total' => intval($tmp[1]), 'used' => intval($tmp[2]));
+    }
+    return $ret; 
+}
+
+function readRaid()
+{
+    $out = array();
+    $ret = array();
+    $out = json_decode(exec('/usr/libexec/nethserver/api/system-status/check-raid'), true);
```

Comment:
If `system-status/check-raid` is not used elsewhere, we can simplify and optimize the process calls:

```suggestion
    $out = json_decode(exec('/usr/libexec/nethserver/raid-status'), true);
```

Reply:
Yes, sure! I will do it. And I will remove check-raid file.